src/scripts/public/accessibility.js

```diff
@@ -20,6 +20,29 @@ const bcgovBlockThemeAccessibility = () => {
             } );
         }
 
+        // Check if the post-content id exists within the document.
+        const postContentExists = qs( '#post-content' );
+
+        // If it doesn't exist, add the id 'post-content' to the mainContent element
+        if ( ! postContentExists ) {
+            // find the most likely place for the content block to begin, starting with the wordpress default of <main>, and falling back.
+            let mainContent = null;
+            const bcgovBodyContentMain = qs( 'main' );
```

Comment:
are you sure this doesn't return an array

Reply:
![image](https://github.com/bcgov/bcgov-wordpress-block-theme/assets/122112933/61b00b60-e37f-433d-8d99-260533c7a3c8)
This returns the first element found

qsa(), or querySelectorAll() should return an array of matching elements.

Reply:
![image](https://github.com/bcgov/bcgov-wordpress-block-theme/assets/122112933/72dedf14-f0da-4864-aa07-d6c348d326cf)
here's an experiment that shows the results